assets/css/v2/bus_eink/evergreen/image.scss

```diff
@@ -0,0 +1,94 @@
+.evergreen-content-image__container {
+  background-color: #e4e6e1;
+  overflow: hidden;
+}
+
+.flex-one-medium-two-small__left,
+.flex-two-medium__left,
+.flex-two-medium__right {
+  .evergreen-content-image__container {
+    border-radius: 16px;
+    box-shadow: 0px 12px 24px 0px rgba(23, 31, 38, 0.25);
```

Comment:
Let's double check with design on whether they want shadows or rounded corners on these in the e-ink apps.

Reply:
Checked with Mindy on this. Border radius on evergreen e-ink is 32px and no drop shadow.